### PR TITLE
added appc_to_ti to the docs

### PR DESCRIPTION
**Pull request by caspahouzer**
added https://github.com/m1ga/from_zero_to_app/blob/master/appc_to_ti.md#cli to the docs as it is often requested in slack

### CODE REVIEW

File: docs/guide/Titanium_SDK/Titanium_SDK_How-tos/README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ weight: '30'
 
 This chapter has several sections dedicated to how-to instructions for Titanium SDK.
 
+* [Transfer your app from appc CLI to ti CLI](/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/)
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_How-tos/README.md]: Minor nitpick, but could you please capitalize and spell out the CLI names:

```
... from Appcelerator CLI to Titanium CLI
```

I think we referring to the CLI's by name, we should use the full name, not the command.

File: docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Transfer your app from `appc` CLI to `ti` CLI
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md]: Use same title as incoming link title. Other than titles, "Appc CLI" should be sufficient.

File: docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## Introduction
+
+Axway announced that they will hand over Titanum to the community at <a href="https://devblog.axway.com/featured/product-update-changes-to-application-development-services-appcelerator/" target="_blank">March 2022</a>. <b>You still create apps or use existing apps!</b>. But you won't have access to the Axway services (e.g. Push, Analytics). You might also see a message like `You cannot perform this operation with your current plan. Please upgrade to build your application for production distribution`. It's time to update the CLI!
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md]: Maybe clean up the wording. The "they will" has already happened.

File: docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+If you want to create a new app you should use the open-source CLI:
+```bash
+ti create
+```
+and follow that menu. Then inside that folder run `alloy new` to convert the classic app to an Alloy app.
+
+## Editor
+
+As of March 2nd 2022 the editor packages will now only use the `ti` and `alloy` CLI's. If your package is still using the `appc` CLI please check for extension updates.
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md]: Again "will" has already happened. FWIW, Ewan and I are going to publish the VSCode extension under the TiDev name soon, so we may want to revisit this paragraph once that's done.

**Review by cb1kenobi — CHANGES_REQUESTED** [review]

File: docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md (hunk 4 above)
Comment (caspahouzer) [reply docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md]: fixed

File: docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md (hunk 3 above)
Comment (caspahouzer) [reply docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md]: fixed

File: docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md (hunk 2 above)
Comment (caspahouzer) [reply docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md]: fixed

File: docs/guide/Titanium_SDK/Titanium_SDK_How-tos/README.md (hunk 1 above)
Comment (caspahouzer) [reply docs/guide/Titanium_SDK/Titanium_SDK_How-tos/README.md]: fixed

File: docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
-# Transfer your app from `appc` CLI to `ti` CLI
+# Transfer your app from Appcelerator CLI to Titanium CLI
 
 ## Introduction
 
-Axway announced that they will hand over Titanum to the community at <a href="https://devblog.axway.com/featured/product-update-changes-to-application-development-services-appcelerator/" target="_blank">March 2022</a>. <b>You still create apps or use existing apps!</b>. But you won't have access to the Axway services (e.g. Push, Analytics). You might also see a message like `You cannot perform this operation with your current plan. Please upgrade to build your application for production distribution`. It's time to update the CLI!
+Axway handed over Titanum to the community at <a href="https://devblog.axway.com/featured/product-update-changes-to-application-development-services-appcelerator/" target="_blank">March 2022</a>. <b>You still create apps or use existing apps!</b>. But you won't have access to the Axway services (e.g. Push, Analytics). You might also see a message like `You cannot perform this operation with your current plan. Please upgrade to build your application for production distribution`. It's time to update the CLI!
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md]: Axway transferred Titanium to the community in ...

Get rid of the `<b>` and replace the `!</b>.` with `,`. You don't want a period to follow a exclamation point.

**Review by cb1kenobi — CHANGES_REQUESTED** [review]

File: docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md (hunk 5 above)
Comment (caspahouzer) [reply docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md]: typos are fixed. thank you

**Review by cb1kenobi — APPROVED** [review]
Looks good!

**Pull request merged**